app/store/page.client.ts

```diff
@@ -0,0 +1,24 @@
+import { persistentAtom } from "@nanostores/persistent";
+
+export type Page = {
+  name?: string;
+  background?: string;
+  weekStart?: string;
```

Comment:
I won't ask you to do it in this PR, but changing this from `string` to `"sunday" | "monday"` is a way to get more benefit from typescript, ensuring you know exactly what kind of strings you'll be working with

Reply:
Makes sense, I had a go at this but was having trouble with type narrowing in setup-page.tsx so I'm going to leave it for now and come back later.

Reply:
Never mind - got it. Had to add a return type.